README.md

```diff
@@ -51,6 +51,7 @@ Install the python. This will also build the TS package.
 
 ```bash
 # First install the python package. This will also build the JS packages.
+yarn install
```

Comment:
Is there a better way to do this?

Reply:
I *think* we just need to use `jlpm` to install the dependencies: https://github.com/jupyter-widgets/widget-ts-cookiecutter/pull/128?